lib/migrator/index.js

```diff
@@ -75,6 +84,7 @@ const migrate = async () => {
   await logger.setup()
 
   try {
+    await preflightChecks(filesToIdentifyPreV13Prototype)
```

Comment:
I think the way the migration script is currently written, this check needs to happen earlier, in `bin/cli`, before we delete the `package.json` file:

https://github.com/alphagov/govuk-prototype-kit/blob/acb7c6c5a6939dfccf0723c1c1d57421a64ea31c/bin/cli#L215

Maybe we should move that code into the `lib/migrator` file 🤔

Reply:
I'll have a look at that now